cves/CVE-2017-5648.yml

```diff
@@ -43,12 +43,15 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
-  While investigating bug 60718, it was noticed that some calls application
-  listeners did not use the appropriate facade object. when running an untrusted
-  application under a security manager, it was possible for that application to
-  retain a references to the request or response objects and thereby acces or
-  modify information associated with another web application.
+description: |
+  While investigating bug 60718, it was noticed that some calls to application
```

Comment:
I would recommend getting less technical in this description, version numbers are not needed and some of the language is very technical.